### PR TITLE
Require slevomat/coding-standard v6

### PR DESCRIPTION
**Pull request by simPod**
This is a base for all possible PRs to come and also for these   
https://github.com/doctrine/coding-standard/pull/138  
https://github.com/doctrine/coding-standard/pull/139

### CODE REVIEW

**Comment by simPod** [comment]
(build fails, those PRs mentioned above are required to pass)

File: composer.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
     "require": {
         "php": "^7.2",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
-        "slevomat/coding-standard": "dev-master#469368cc6d8fe83aceba259ef65f1c0a2f63055b",
-        "squizlabs/php_codesniffer": "^3.5.0"
+        "slevomat/coding-standard": "^6.0",
+        "squizlabs/php_codesniffer": "^3.5"
```
Comment (carusogabriel) [inline composer.json]: Please lock at 
```suggestion
        "squizlabs/php_codesniffer": "^3.5.3"
```

Comment (simPod) [reply composer.json]: Ok. I though it's sufficient that 3.5.3 is resolved through slevomat/cs and for us 3.5 will generally do.

**Comment by simPod** [comment]
Superseded by https://github.com/doctrine/coding-standard/pull/144

**Pull request closed without merging**